tests/test_visualization.py

```diff
@@ -142,6 +144,10 @@ def test_get_contour_plot():
     assert figure.layout['xaxis']['range'] == (1.0, 2.5)
     assert figure.layout['yaxis']['range'] == (0.0, 2.0)
 
+    # Test ValueError due to wrong params
```

Comment:
Please add a period at the end of the sentence. (You may want to fix the other comments as well in this method.)